content/knowledge_base/flare/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,30 @@
+# Avalanche Consensus Algorithm Concepts
+
+## Snow Algorithm Group of Protocols
+
+A group of protocols with a strong probabilistic safety guarantee in the presence of Byzantine nodes.
+
+### Description
+
+The core concept of operating is execution of repeated sampling the network at random, ultimately leading to correct nodes behavior to obtain a common statement (decision, transacation value, outcome).
+This mechanism effectively brings the system to an irreversible state — meaning that a large potion of the network accepted a decision or a statement.
+So any such conflicting to that statement would be accepted with a negligible probability ε or smaller.
+TODO: finish
+A more thorough description of Snow Algorithms specifications and examples can be found in [Avalanche Whitepaper](https://assets-global.website-files.com/5d80307810123f5ffbb34d6e/6009805681b416f34dcae012_Avalanche%20Consensus%20Whitepaper.pdf)
```

Comment:
This has been fixed here https://github.com/optakt/optakt.github.io/pull/13/commits/3495fab15d48fd58092b6396188990fc20e2177c#diff-e77cfdf99e3d26834c46501127cb0279d60366b93a3afffc6c90fcd65286b6a4R13-R14
Thanks!
Added a new line, because it seemed to make sense to separate the link.